tsconfig.json

```
@@ -30,6 +30,6 @@
   },
   "include": [
     "./src/**/*.ts",
-    "src/index.tsx",
+    "src/index.tsx", "src/lib/useCsvReactPlugin.tsx",
```

you can add general one 
```suggestion
    "src/index.tsx", "./src/**/*.tsx",
```